libraries/dagster-contrib-weaviate/dagster_contrib_weaviate/config.py

```diff
@@ -0,0 +1,27 @@
+from typing import Literal
+
+from dagster import Config
+
+class LocalConfig(Config):
+    """Connection parameters for a local (self-hosted) database."""
+    
+    provider: Literal["local"] = "local"
+
+    host: str = "localhost"
+    """The host to use for the underlying REST and GraphQL API calls."""
+
+    port: int = 8080
+    """The port to use for the underlying REST and GraphQL API calls."""
+
+    grpc_port: int = 50051
```

Comment:
where do these defaults come from?

At the very least, I would like them to be set as constants and comment where they're coming from